x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -527,6 +528,21 @@ export const uiSettings: Record<string, UiSettings> = {
       }
     ),
   },
+  [profilingAzureCostDiscountRate]: {
+    category: [observabilityFeatureId],
+    name: i18n.translate('xpack.observability.profilingAzureCostDiscountRateUiSettingName', {
+      defaultMessage: 'Azure discount rate (%)',
+    }),
+    value: 6,
```

Comment:
Q: Why is the value `6` here? Should it be 0 by default 🤔

Reply:
This is still an open question. I'll change it as soon I get it.